src/community/design-system-day-2024/index.md

```diff
@@ -13,9 +13,7 @@ order: 15
 
 ## Join us for a Design System "Hack" Day
```

Comment:
`## Join us for a Design System 'Hack' Day`

Reply:
Done!